__tests__/account.test.ts

```diff
@@ -65,6 +65,40 @@ describe('deploy and test Wallet', () => {
     innerInvokeEstFeeSpy.mockClear();
   });
 
+  test('estimate fee bulk', async () => {
+    const res = await account.estimateFeeBulk([
+      {
+        type: 'DECLARE',
```

Comment:
@irisdv I suggest that on top of this "generic" test, we add a test for the main scenario this bulk endpoint caters which is: 
Txn #1: DEPLOY_ACCOUNT
Txn #2: MULTI_CALL (of INVOKE-s)

Reply:
When adding this test I realised I have an issue when estimating fees for DEPLOY_ACCOUNT. I have this error when I’m trying to use the function `estimateAccountDeployFee` on its own : `Entry point 0x36fcbf06cd96843058359e1a75928beacfac10727dab22a3972f0af8aa92895 not found in contract with class hash 0x3fcbf77b28c96f4f2fb5bd2d176ab083a12a5e123adeb0de955d7ee228c9854`

Is there a specific way to make it work ? My understanding is that to deploy a new account I need to declare it, send eth to the address then deploy it.

Reply:
The `0x36fcbf06cd96843058359e1a75928beacfac10727dab22a3972f0af8aa92895` selector is `__validate_deploy__` meaning that the contract you tried to deploy does not implement this endpoint and thus `DEPLOY_ACCOUNT` fails. In order for it to work, you need to implement/use a contract that have the `__validate_deploy__` entry-point - See example in OpenZeppelin's account contract:
https://github.com/OpenZeppelin/cairo-contracts/blob/e0e07555b319d8131a35259b55269ba7b19e744e/src/openzeppelin/account/presets/Account.cairo#L112

Reply:
Thanks, I updated the version of Account.json in the repo and added this testcase.